### PR TITLE
Service monitor per dbin

### PR DESCRIPTION
**Pull request by allanger**
Instead of creating one service monitor for the whole chart, I'm creating a service monitor per DbIn with `.serviceMonitor.enabled` 

Also, I've added a helm test with ci, so we can check that the data from the pg_exporter is scraped by prometheus, I don't test the logic of scraping, only that there is something from the exporter, to see if ServiceMonitor is working

It's a major version bump, cause it's a breaking change.

### CODE REVIEW

**Review by dabde — COMMENTED** [review]
some comments.

File: charts/db-instances/files/test_service_monitor [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    exit 1
+  else 
+    TEST_PASSED=true
+    break
+  fi
+  sleep "${ATTEMPTS_PAUSE}"
+done
+
+if [ "${TEST_PASSED}" != "true" ]; then
+  echo "[ERROR]: Test is failed, please check the logsd"
```
Comment (dabde) [inline charts/db-instances/files/test_service_monitor:49]: typo `logsd`.

File: charts/db-instances/files/test_service_monitor [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  # -- for some reason this jq query only works with pips in the pod
+  METRIC_NAME=$(curl -q "${QUERY_URL}" | jq -r '.data.result | .[] | .metric.__name__')
+  if [ "${METRIC_NAME}" != "${QUERY}" ]
+  then
+    echo "query didn't return expected name"
+    echo "${METRIC_NAME}"
+    exit 1
+  else 
+    TEST_PASSED=true
+    break
```
Comment (dabde) [inline charts/db-instances/files/test_service_monitor:43]: you could make this to `exit 0`, then you don't need the `TEST_PASSED` variable and remove the if condition in the end.

(minor)

**Review by dabde — COMMENTED** [review]
minor comments.

File: charts/db-instances/files/test_service_monitor (hunk 1 above)
Comment (allanger) [reply charts/db-instances/files/test_service_monitor]: Should be fixed

**Review by dabde — APPROVED** [review]

File: charts/db-instances/files/test_service_monitor (hunk 2 above)
Comment (allanger) [reply charts/db-instances/files/test_service_monitor]: I'm closing the conversation,since the PR is approved

**Pull request merged**